utils/hlc/src/main.cpp

```diff
@@ -97,23 +97,6 @@ int main(int argc, const char **argv) {
 
         entire_flow_end = clock();
 
-        auto& timing_ctx = g_vpr_ctx.timing();
```

Comment:
Not sure why we are removing this, but also don't see the value of the keeping it.

Reply:
It doesn't compile, and like you said there is little value to keeping it.  When I wrote genfasm, mithro asked me to remove it, so this is actually more consistent anyways.